style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+body {
+  max-width: 375px;
```

Comment:
[OPTIONAL] I would suggest you remove the max-width of your body tag when you start working on the desktop version of your page in future milestones. You can always use the dev tools of your web browser to resize the window to mobile sizes and check that your mobile version is still looking great!